content/documentation/staging/installation-guide/index.adoc

```diff
@@ -253,6 +265,63 @@ icon:lightbulb[size=1x]{nbsp} If you installed the Kiali CR in a different names
 
 Note that even if this fixes the hang problem, you may still have remnants of the Kiali Server in your cluster. You will manually need to delete those resources.
 
+== Upgrade
+
+=== Upgrading Istio
+
+In general, Kiali tries to maintain compatability between at least the two most recent minor versions of Istio e.g. Kiali 1.26.0 is compatible with both Istio 1.7 and 1.8. If you are upgrading one minor version of Istio, before upgrading Istio, upgrade Kiali to the most recent version that supports that version of Istio. See the link:#_version_compatibility[version compatability] section for more details on which versions are supported. 
+
+==== Canary
+
+For https://istio.io/latest/docs/setup/upgrade/canary/[Canary upgrades]:
+
+. Deploy the upgraded https://istio.io/latest/docs/setup/upgrade/canary/#control-plane[istio control-plane].
+. Update the Kiali Server configuration to point to the new Istio deployment. These fields need to be updated:
+- `spec.external_services.istio.config_map_name` to the new Istio configmap revision.
+- `spec.external_services.istio.istiod_deployment_name` to the new istio deployment revision.
+- `spec.external_services.istio.istio_sidecar_injector_config_map_name` to the new istio sidecar injector configmap revision.
+
+If you are using the *kiali-operator*, update the Kiali CR configuration:
+
+[source,bash]
+----
+  kubectl patch kialis kiali -n kiali-operator \
+    -p '{"spec": {"external_services": {"istio": {"config_map_name": "istio-canary", "istiod_deployment_name": "istiod-canary", "istio_sidecar_injector_config_map_name": "istio-sidecar-injector-canary"}}}}' \
+    --type=merge
+----
+
+Wait until the operator restarts the Kiali Server and then verify everything is working correctly.
+
+If you are using the `kiali-server` *Helm Chart*, set the Helm Chart values: 
+
+[source,bash]
+----
+  helm upgrade \
+    --set external_services.istio.config_map_name=istio-canary \
+    --set external_services.istio.istio_sidecar_injector_config_map_name=istio-sidecar-injector-canary \
+    --set external_services.istio.istiod_deployment_name=istiod-canary \
+    --repo https://kiali.org/helm-charts \
+    --namespace istio-system \
+    kiali-server kiali-server
+----
+
+Restart the Kiali pod to pickup the new configmap changes.
```

Comment:
I don't know how to make this flow better, but I can only tell you what I first thought when I read this. I thought this was saying you should restart the pod regardless of whether you were using the server helm chart or operator.  Now, I know the answer :) but the flow of this made it ambiguous. Do I restart the pod even if I'm using the operator? It isn't really clear (though, yes, this appears after the sentence "If you are using the kiali-server".... but because there are no clear delineated sections here, it seems ambiguous.)

Perhaps all we need to do is change this sentence to be:
```suggestion
If you are using the `kiali-server` helm chart, you must restart the Kiali pod in order to pickup the new ConfigMap changes. This is not required if you are using Kiali Operator.
```

Reply:
That's good feedback. I originally had *kiali-operator* and *Helm Chart* as subsection headers underneath the Canary section. Do you think that'd make it clearer? Something like

### Canary
For Canary Upgrades ...

#### Kiali Operator
If you are using the *kiali-operator*, update the Kiali CR configuration:
...

#### Helm Chart - Kiali Server
If you are using the `kiali-server` *Helm Chart*, set the Helm Chart values: 
...

I also wonder if it'd be helpful to call the kiali server helm chart deployment something like the "standalone" deployment or if that's more confusing?

Reply:
I was thinking having separate sections like that. That's fine.

"standalone" deployment would be more confusing. I would keep using "Kiali Server Helm Chart" to refer to this.